### PR TITLE
added the update functionality

### LINKED ISSUE #30: feat: add update blog functionality in frontend.
Users should be able to either update the title or the content of the blog.

### PR DESCRIPTION
**Pull request by Ameerjafar**
I have added the update functionality.

https://github.com/user-attachments/assets/79c152e6-e106-4979-aa4c-7d41d57bc455

fix: #30

### CODE REVIEW

**Comment by Ameerjafar** [comment]
@geekyharsh05 can you kindly review the pr

**Comment by geekyharsh05** [comment]
@Ameerjafar Can you please adjust it according to the dark mode and there is a typo in the submit button it is written as `Sumbit`.
<br />
Screenshot 2024-10-14 at 6 55 19 PM

File: frontend/src/routes/BlogView.tsx [hunk 1]
```diff
@@ -85,9 +115,30 @@ const ViewBlog = () => {
...
+            ): (
+                <h1 className="text-3xl sm:text-4xl font-bold mb-4 text-gray-900 dark:text-gray-100">
+                {post.title}
+              </h1>
+            )
+
+            }
+            {isOpen ? (
+              <div className = 'pl-52 pt-2'>
+                <button onClick = { SumbitHandler } className = 'w-24 h-10 bg-blue-500 rounded-md  text-white'>Sumbit</button>
```
Comment (geekyharsh05) [inline frontend/src/routes/BlogView.tsx]: There's a typo in the button name.

**Review by geekyharsh05 — CHANGES_REQUESTED** [review]

**Comment by Ameerjafar** [comment]
@geekyharsh05 i have made the changes

**Review by geekyharsh05 — APPROVED** [review]

**Comment by geekyharsh05** [comment]
@Ameerjafar Thanks for your contribution!

**Pull request merged**